share/spice/drinks/drinks.js

```diff
@@ -0,0 +1,46 @@
+(function (env) {
+    "use strict";
+
+    function getInfoBoxData(item) {
+        var infoboxData = [{
+            heading: 'Ingredients:'
+        }];
+        for (var i = 1; i <= 15; i++) {
+            if(item["strIngredient" + i] !== "") {
+                infoboxData.push({
+                    label: item["strMeasure" + i] + "" + item["strIngredient" + i]
+                });
+            }
+        }
+        return infoboxData;
+    }
+
+    env.ddg_spice_drinks = function(api_result){
+
+        if (!api_result || api_result.error) {
```

Comment:
Please check if `drinks` is `null`:

![screen shot 2015-09-28 at 4 57 03 pm](https://cloud.githubusercontent.com/assets/81969/10148524/0741393e-6602-11e5-8ed2-e05a4993e9fe.png)